tpdatasrc/co8fixes/scr/py00281Yvy.py

```diff
@@ -0,0 +1,23 @@
+from toee import *
```

Comment:
In principle this should also be in Co8infra, however KotB uses a different script name... blargh
So I guess IWD with far shot will be kinda broken. Unless someone alters the script there ;)

Reply:
No fix there then.